contracts/integrations/examples/contractCreator/withoutTreasury/SingleExecTaskCreator.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../../OpsTaskCreator.sol";
+
+/**
+ * @dev
+ * Example contract that creates a single exec task.
+ */
+// solhint-disable not-rely-on-time
+// solhint-disable no-empty-blocks
+contract SingleExecTaskCreator is OpsTaskCreator {
+    uint256 public count;
+    uint256 public lastExecuted;
+    bytes32 public taskId;
+    uint256 public constant MAX_COUNT = 5;
+    uint256 public constant INTERVAL = 3 minutes;
+
+    event CounterTaskCreated(bytes32 taskId);
+
+    constructor(address payable _ops, address _owner)
+        OpsTaskCreator(_ops, _owner)
+    {}
+
+    function createTask() external payable {
+        require(taskId == bytes32(""), "Already started task");
+
+        bytes memory execData = abi.encodeCall(this.increaseCount, (1));
+
+        ModuleData memory moduleData = ModuleData({
+            modules: new Module[](2),
+            args: new bytes[](2)
+        });
+
+        moduleData.modules[0] = Module.PROXY;
+        moduleData.modules[1] = Module.SINGLE_EXEC;
+
+        moduleData.args[0] = _proxyModuleArg();
+        moduleData.args[1] = _singleExecModuleArg();
+
+        bytes32 id = ops.createTask(
+            address(this),
+            execData,
+            moduleData,
+            address(0)
```

Comment:
shouldn't this one be `ETH` for task without treasury?
```suggestion
            ETH
```

Reply:
[1a637b0](https://github.com/gelatodigital/ops/pull/88/commits/1a637b04ec8568e7dd0d0044839aba42a69022e5)